src/store/store.ts

```diff
@@ -523,15 +486,13 @@ export class Store extends EventTarget {
     const emittedSet = new Set<string>();
 
     for await (const [entry] of willowQuery) {
-      const path = willowToEarthstarPath(entry.path);
-      const joined = path.join("/");
+      const path = new Path(entry.path);
+      const formatted = path.format("ascii");
```

Comment:
If we have encrypted entries in the store, they likely can't be decoded as ascii, so `queryPaths` stops working

Reply:
Good catch, thank you!